source/opt/ir_context.h

```diff
@@ -418,6 +427,11 @@ class IRContext {
   // Returns true if all of the given analyses are valid.
   bool AreAnalysesValid(Analysis set) { return (set & valid_analyses_) == set; }
 
+  // Returns true if all of the given analyses are valid.
```

Comment:
Wrong comment I guess. You mean all the given analyses are disabled?

Reply:
Yeah. Cut and paste error.